packages/solana/src/utils/chainPath/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const SolanaChainIDs = {
+	Mainnet: 'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp',
```

Comment:
What about the testnet? Don't we have to include it as well? Or maybe not any changes to it? @glitch-txs

Reply:
testnet ID was added last so no need